src/acl.c

```diff
@@ -1159,6 +1220,58 @@ int ACLCheckCommandPerm(client *c, int *keyidxptr) {
     return ACL_OK;
 }
 
+/* Check if the pub/sub channels of the command, that's ready to be executed in
+ * the client 'c' and already referenced by c->cmd, can be executed by this
```

Comment:
c->cmd is a bit of a red herring here based on the implementation.

Reply:
Red herring is no more.